### PR TITLE
New version Viber.Viber version 20.5.1.2

### PR DESCRIPTION
**Pull request by mrjcap**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/112809&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/v/Viber/Viber/20.5.1.2/Viber.Viber.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://download.cdn.viber.com/desktop/windows/ViberSetup.msi
+  InstallerSha256: 121B729EAA36DD5D20BB2044E2543B60FBEAE99F3EA208A1720CE2FF03C7FA98
+  ProductCode: '{AD5595BB-39D2-454D-A6E5-7775FCDF1C33}'
```
Comment (ItzLevvie) [inline manifests/v/Viber/Viber/20.5.1.2/Viber.Viber.installer.yaml:19]:
```suggestion
  ProductCode: '{EE9DED09-83F6-400F-BCE5-960E87D3D2BA}'
```
```
Name          : Viber
Publisher     : 2010-2023 Viber Media S.a.r.l
Version       : 20.5.1.2
ProductCode   : {EE9DED09-83F6-400F-BCE5-960E87D3D2BA}
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\{EE9DED09-83F6-400F-BCE5-960E87D3D2BA}
Uninstall     : winget uninstall --id "{EE9DED09-83F6-400F-BCE5-960E87D3D2BA}"
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by mrjcap** [comment]
What's wrong with this one?

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> ##[error] Installer https://download.cdn.viber.com/desktop/windows/ViberSetup.msi
Scan request id: 'b1f531f5-2451-4d69-b7e2-ae3484101029'
Error information: BlockingDetectionFound
##[error] Result: Failed

**Comment by stephengillie** [comment]
During install, this package caused this error almost 9000 times:
> 9:17:00 AM: [21F4:1FB4][2023-07-24T09:12:41]e000: Error 0x8007139f: Application requested to send embedded progress message when not in embedded mode.

**Comment by mrjcap** [comment]
I changed the command line switch on the msi installer. Same error. I have no problem in the Windows Sandbox. 

Screenshot:

image

**Comment by stephengillie** [comment]
> I changed the command line switch on the msi installer. Same error. I have no problem in the Windows Sandbox.

Which error are you seeing - the product code? 

Windows Sandbox omits several foundational parts of the OS - it works about the same as a full Windows install on about 80% of packages.

**Comment by mrjcap** [comment]
The first one you mentioned earlier:

##[error] Installer https://download.cdn.viber.com/desktop/windows/ViberSetup.msi
Scan request id: '3ebf19b4-ba85-420f-ac93-973e088ccb22'
Error information: BlockingDetectionFound
##[error] Result: Failed

**Comment by stephengillie** [comment]
This might be due to the Sandbox not having Defender.

**Comment by mrjcap** [comment]
I'll go ahead and close this, since a new version is out.

**Pull request closed without merging**